jpa/eclipselink.jpa.test/src/org/eclipse/persistence/testing/tests/jpa/beanvalidation/BeanValidationJunitTest.java

```diff
@@ -355,7 +366,34 @@ public void testTraversableResolverPreventsTraversingRelationshipMultipleTimes()
         // If it ever finds an entity with such flag set, the entity has visited the validator twice. It should be flagged as error.
     }
 
-
+    //Bug #411013
+    public void testValidateChangedData() {
+        try {
+            getDatabaseSession().executeNonSelectingSQL("insert into CMP3_BV_PROJECT values (895, \"some long name\")");
+        } catch (Throwable t) {
+            //ignore
```

Comment:
If the INSERT fails (other than a duplicate key exception), doesn't that throw off the rest of the test?

Reply:
Failed 'testSetup' would lead to the same situation..